test/optimus/pr/test_inputs.json

```diff
@@ -4,20 +4,20 @@
   "TestOptimusPR.expected_matrix_summary_hash": "dd513351d4e7688c97f7bf902ba2876f",
   "TestOptimusPR.expected_picard_metrics_hash": "7b7be5c9a2236920ca09f05811dca6d5",
   "TestOptimusPR.r1": [
-    "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
-    "gs://broad-dsde-mint-dev-teststorage/10x/demo/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz"
+    "gs://hca-dcp-mint-test-data/optimus/fastqs/pbmc8k_S1_L007_R1_001.fastq.gz",
```

Comment:
I think the test inputs should be the same as the inputs here for consistency: https://github.com/HumanCellAtlas/skylab/pull/84/files#diff-7787cfa08c74e3392fe187d33f67dca1R4, although I'm not sure whether we want to use the data in the Optimus or 10x folder (I didn't realize there were both)!